logging/cluster-logging-release-notes.adoc

```diff
@@ -1,6 +1,8 @@
-[id="cluster-logging-release-notes"]
-= Release notes for Red Hat OpenShift Logging 5.4
+:_content-type: ASSEMBLY
+[id="cluster-logging-release-notes_{context}"]
```

Comment:
```suggestion
[id="cluster-logging-release-notes"]
```
Assembly files don't have a parent document from which to inherit a context value. I also spot-checked a few assemblies to make sure they didn't have this context attribute: they don't.